stable/artifactory-ha/templates/_helpers.tpl

```diff
@@ -10,14 +10,16 @@ Expand the name of the chart.
 The primary node name
 */}}
 {{- define "artifactory-ha.primary.name" -}}
-{{- printf "%s-%s-primary" .Release.Name .Chart.Name | trunc 63 | trimSuffix "-" -}}
+{{- $name := .Release.Name | trunc 15 -}}
```

Comment:
to be sure we do have not too short release name, change to 29, tested it myself